### DISCUSSION TITLE
사운드 사용처

### DISCUSSION THREAD

**Discussion by seong0929** [SeoulTechTCPGame/HumanSurvival#249] [Q&A]
- 기본 BGM
- 인 게임 스테이즈 BGM
- 무기 11개

**Comment by seong0929** [comment]
이 외에 사용되는 사운드가 있을까요?
e.g. 피격, 타격, 게임 오버 BGM, 임펙트, 플레이어 이동, 플레이어 죽음, 몬스터 죽음 등

**Comment by seong0929** [comment]
Whip: 0, 채찍
MagicWand: 1, 마법 지팡이
Knife: 2, 단검
Cross: 3, 십자가
KingBible: 4, 성경
FireWand: 5, 불의 지팡이
Garlic: 6, 마늘
Peachone: 7, 하얀 비둘기
EbonyWings: 8, 검은 비둘기
LightningRing: 9, 번개 반지

**Comment by rapa1130** [comment]
## 사운드 사용처 정리
**레벨업 효과음**을 제외한 다른 사운드들은 [사운드 링크](https://assetstore.unity.com/packages/audio/sound-fx/free-retro-sfx-pack-43256#content) 에서 찾을 수 있음

**레벨업 효과음**은 [레벨업 사운드 링크](https://assetstore.unity.com/packages/audio/sound-fx/rpg-essentials-sound-effects-free-227708)에서 찾을 수 있음

아래는 각 사용처별 사용이 추천되는 사운드 파일 이름임.

### 몬스터 타격음
Attack
### 캐릭터 피격음
Die_2
### 아이템 먹는 소리(다른 에셋임을 유의!!)
Jump4
### 레벨업 효과음
48_Speed_Up
### UI 선택음
Click
### UI 취소음
Pop_error
### 플레이어 죽음 효과음
Die_3

**Comment by seong0929** [comment]
### 필요
- 보물 상자 얻을 때 나오는 효과음
- 보물 상자가 열리는 도중에 나오는 효과음
